### ISSUE TITLE
Breast Cancer Histopathological Image Classification: A Deep Learning Approach

### ISSUE THREAD

**Issue by SiminaB** [greenelab/deep-review#748]
https://doi.org/10.1101/242818 https://www.biorxiv.org/content/early/2018/01/04/242818

> Breast cancer remains the most common type of cancer and the leading cause of cancer-induced mortality among women with 2.4 million new cases diagnosed and 523,000 deaths per year. Historically, a diagnosis has been initially performed using clinical screening followed by histopathological analysis. Automated classification of cancers using histopathological images is a chciteallenging task of accurate detection of tumor sub-types. This process could be facilitated by machine learning approaches, which may be more reliable and economical compared to conventional methods. To prove this principle, we applied fine-tuned pre-trained deep neural networks. To test the approach we first classify different cancer types using 6,402 tissue microarrays (TMAs) training samples. Our framework accurately detected on average 99.8% of the four cancer types including breast, bladder, lung and lymphoma using the ResNet V1 50 pre-trained model. Then, for classification of breast cancer sub-types, this approach was applied to 7,909 images from the BreakHis database. In the next step, ResNet V1 152 classified benign and malignant breast cancers with an accuracy of 98.7%. In addition, ResNet V1 50 and ResNet V1 152 categorized either benign- (adenosis, fibroadenoma, phyllodes tumor, and tubular adenoma) or malignant- (ductal carcinoma, lobular carcinoma, mucinous carcinoma, and papillary carcinoma) sub-types with 94.8% and 96.4% accuracy, respectively. The confusion matrices revealed high sensitivity values of 1, 0.995 and 0.993 for cancer types, as well as malignant- and benign sub-types respectively. The areas under the curve (AUC) scores were 0.996,0.973 and 0.996 for cancer types, malignant and benign sub-types, respectively. Overall, our results show negligible false negative (on average 3.7 samples) and false positive (on average 2 samples) results among different models. Availability: Source codes, guidelines, and data sets are temporarily available on google drive upon request before moving to a permanent GitHub repository.

**Comment by zhanghn2012** [comment]
Dear SiminaB,

Would you like share the codes and datasets in GitHub repository or other places? If you would like to, that will help a lot because I am doing some acedemic research related to breast cancer. But I meet some coding problems and have no idea about how to detect breast cancer with deeping learning methods. If you can help, I will appreciate it! Looking forward to hearing from you.

**Comment by agitter** [comment]
Hi @zhanghn2012, @SiminaB and the others who are active in this GitHub repository are not directly involved in the study linked above.  We are discussing deep learning manuscripts for a review article.  The authors' email addresses are available from https://www.biorxiv.org/content/early/2018/01/04/242818.article-info

**Comment by stephenra** [comment]
@am-sirdaniel This repository has no affiliation with the original study. Please read the thread above.

>Hi @zhanghn2012 , @SiminaB and the others who are active in this GitHub repository are not directly involved in the study linked above. We are discussing deep learning manuscripts for a review article. The authors' email addresses are available from https://www.biorxiv.org/content/early/2018/01/04/242818.article-info
